tests/test_simpleflow/test_dataflow.py

```diff
@@ -40,6 +39,9 @@
 )
 
 
+Executor.decref_workflow = lambda _: None  # FIXME use @patch?
```

Comment:
I'm fine with that personally ;-) Maybe add just a tiny comment to explain we want to access the workflow instance in the executor for test purposes?